src/ui/controllers/booking.ts

```diff
@@ -0,0 +1,168 @@
+import * as express from 'express'
+import * as template from 'lib/ui/template'
+import * as courseController from './course/index'
+import * as catalog from 'lib/service/catalog'
+import * as model from 'lib/model'
+import * as dateTime from 'lib/datetime'
+
+export async function renderBookableCourseInformation(
```

Comment:
As discussed this will/should happen in courseController, keep this controller for the actual booking process. I'd drop the breadcrumbs from the overview page for now too as they're not on any other overview page.